libusb/os/darwin_usb.c

```diff
@@ -1586,12 +1587,34 @@ static int darwin_release_interface(struct libusb_device_handle *dev_handle, uin
   return darwin_to_libusb (kresult);
 }
 
+static int darwin_clear_check_alt_setting_and_clear_halt(struct libusb_device_handle *dev_handle, uint8_t altsetting, struct darwin_interface *cInterface) {
```

Comment:
Naming: Is the first _clear_ intentional?
And do we need darwin_ on all static functions?

Reply:
Fixed. Was not matching my own naming conventions. The clear was not intentional